docs/dashboards/multiple-series.md

```diff
@@ -6,96 +6,86 @@ redirect_from:
 
 # Charts with multiple series
 
-Data in isolation is rarely all that useful. One of the best ways to add context and clarity when communicating with data is to show data side-by-side with other data. Here are just a few examples of data that is better together than apart.
+One of the best ways to add context and clarity when communicating with data is to show data side-by-side with other data. For example:
 
-- Your company’s revenue vs. its costs over time.
-- Average order price this month and user signups for that month.
-- New users per day vs. returning users per day.
-- Orders per day from a few different product lines.
+- Your company's revenue vs. its costs over time
+- Average order price this month and user signups for that month
+- Orders per day from a few different product lines
 
 ## Displaying data side by side
 
-There are two main ways to visualize data side by side:
+There are two main ways to visualize data side-by-side in Metabase:
 
-1. [**Ask a question that involves multiple dimensions**](#ask-a-question-that-involves-multiple-dimensions) with the query builder (or in SQL, if you’re fancy). Example: the count of users by region over time.
-
-2. [**Combine two saved questions**](#combining-two-saved-questions) that share a common dimension (like time) on a dashboard. For example, you could look at revenue over time and costs over time together.
+- [**Ask a question that involves multiple dimensions**](#ask-a-question-that-involves-multiple-dimensions) with the query builder (or in SQL, if you're using SQL). For example, the count of users by region over time.
+- [**Combine multiple questions on one dashboard card**](#combining-multiple-questions-on-one-dashboard-card) that share a common dimension (like time) on a dashboard. For example, you could look at revenue over time and costs over time together.
 
 ## Ask a question that involves multiple dimensions
 
-If you’re creating a new question, you can view the results as a multi-series visualization by summarizing your data and grouping it into two groups.
-
-As an example, we might want to see which website or service is referring the most people to our website. In the **Sample Database** that ships with Metabase, you would group by the `Source` and `Created At` columns of the **People** table.
+If you're creating a new question, you can view the results as a multi-series visualization by summarizing your data and grouping it into two or more groups.
```

Comment:
```suggestion
If you're creating a new question, you can view the results as a multi-series visualization by summarizing your data and grouping it into two or more groups.
```
incorrect - not into two or more groups, but using two or more dimensions. ("Widget" and "Gadget" are groups; "Category" and "Created At" are dimensions)